wbooks/src/redux/books/reducer.ts

```diff
@@ -1,14 +1,14 @@
 import { actions } from './actions';
 
 const initialState = {
-  book: null
+  books: null
 };
 
 export default function detailBookReducer(state = initialState, action: any) {
   switch (action.type) {
-    case actions.DETAIL_BOOK:
+    case actions.GET_BOOKS_SUCCESS:
```

Comment:
Let's add the case for GET_BOOKS and GET_BOOKS_FAILURE.
GET_BOOKS should set the loading as true.
SUCCESS and FAILURE should set the loading and false.

Also, add a simple https://reactnative.dev/docs/activityindicator in the books component if the loading is set to true